package.json

```diff
@@ -71,7 +70,8 @@
     "test:headless": "node ./tests/headless.js",
     "test:karma": "npm run karma",
     "prepare": "npm run snyk-protect",
-    "prepublish": "npm run snyk-protect"
+    "prepublish": "npm run snyk-protect",
+    "parcel": "parcel  ./builds/respec-w3c-common.js -d ./builds/output"
```

Comment:
`build` is omitted: `"parcel build ./builds/respec-w3c-common.js -d ./builds/output"`

Reply:
I Have tried adding build before but is repeatedly throws this error:
```
parcel build ./builds/respec-w3c-common.js -d ./builds/output

🚨  /home/nilesh/respec/builds/respec-w3c-common.js: sym.definition is not a function
    at may_modify (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:8931:31)
    at TreeTransformer.abort [as before] (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:8541:74)
    at AST_VarDef.eval [as transform] (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:4570:35)
    at eval (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:4586:25)
    at doit (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:130:23)
    at MAP (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:156:52)
    at do_list (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:4585:16)
    at eval (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:4664:28)
    at AST_Const.eval [as transform] (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:4573:17)
    at collapse (eval at <anonymous> (/home/nilesh/respec/node_modules/uglify-es/tools/node.js:21:1), <anonymous>:8617:39)
```

Reply:
Ah yes, this is the uglify-es problem we met in #1489 that will be fixed in its next release. See https://github.com/mishoo/UglifyJS2/issues/2896

Reply:
Let's do not use it until new release ,Using `--no-minify` removes this error for me. https://github.com/parcel-bundler/parcel/issues/8. What do you think @saschanaz  @marcoscaceres ?

Reply:
We ultimately want to minify this, otherwise the bundle size will be huge... so we are kinda stuck until the next Uglify release.

Reply:
`--no-minify` may still be good for debugging purpose, where source mapping is frequently not perfect.